src/com/sfuentes/OnlyPositiveTest.java

```diff
@@ -0,0 +1,81 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class OnlyPositiveTest {
+  public static void main(String[] args) {
+    showOnlyPositiveNumbers_positiveIsCorrect();
+    showOnlyPositiveNumbers_Zero();
+    showOnlyPositiveNumbers_Negative();
+    showOnlyPositiveNumbers_sameNumbersPositive();
+  }
+
+  public static void showOnlyPositiveNumbers_positiveIsCorrect() {
+    OnlyPositive correct = new OnlyPositive();
+
+    double[] numbers = {1.5, 0, -9, 7.8, -4, -3, -2, -8, -9, -10};
+    ArrayList<Double> expectedResult = new ArrayList<>(Arrays.asList(1.5d, 7.8d));
+    ArrayList<Double> actualResult = correct.showOnlyPositiveNumbers(numbers);
+
+    System.out.println("Test case 1: Show positive numbers correct: \n\t" +
+        "Test numbers: " + Arrays.toString(numbers) + " \n\t" +
+        "Expected result: " + expectedResult + " \n\t" +
+        "Actual result: " + actualResult);
+    if (actualResult.equals(expectedResult))
+      System.out.println("\tSUCCESSFUL TEST.");
+    else
+      System.out.println("\tTEST FAILED.");
+  }
+
+  public static void showOnlyPositiveNumbers_Zero() {
+    OnlyPositive zero = new OnlyPositive();
+
+    double[] numbers2 = {0, 0, 0, 0, 0};
+    ArrayList<Double> expectedResult2 = new ArrayList<>(Arrays.asList());
```

Comment:
remove the suffix "2" from your variable is not needed. Do it in all other tests.

Reply:
Fixed it